Tests/SubprocessTests/SubprocessTests.swift

```diff
@@ -29,10 +33,34 @@ final class SubprocessTests: XCTestCase {
             executing: .named("curl"),
```

Comment:
doesn't this test require internet? There are enough local utilities to test with that this should be unnecessary